yazi-config/preset/yazi.toml

```diff
@@ -9,10 +9,12 @@ show_hidden    = false
 show_symlink   = true
 
 [preview]
-tab_size   = 2
-max_width  = 600
-max_height = 900
-cache_dir  = ""
+tab_size        = 2
+max_width       = 600
+max_height      = 900
+cache_dir       = ""
+ueberzug_scale  = 1
```

Comment:
I guess you want to keep all comments of the config in your website, which is ok for users but inconvenient for developers. Without context I have no way to tell the usage of `ueberzug_scale` and `ueberzug_offset`, as their name is not that intuitive, especially `ueberzug_offset` is an unnamed tuple. I'd suggest have comments somewhere in the code and keep them in sync with your doc.

Reply:
The configuration for Yazi is currently centralized on a standalone website.

This is primarily done to avoid maintaining multiple copies of documentation simultaneously. Maybe in the future, we can explore generating documentation from the code, but for now, let's keep this approach.